component/notification/src/main/java/org/exoplatform/social/notification/LinkProviderUtils.java

```diff
@@ -306,4 +306,14 @@ public static String getOpenLink(ExoSocialActivity activity) {
     }
     return null;
   }
+
+  /**
+   * Get the Redirect quarantine page url
+   *
+   * @return url http://localhost:8080/portal/login?initialURI=/portal/g/:platform:administrators/dlp-quarantine
+   */
+  public static String getQuarantineRedirectURL() {
+    String portal = PortalContainer.getCurrentPortalContainerName();
+    return new StringBuffer(CommonsUtils.getCurrentDomain()).append("/").append(portal).append(LOGIN_INITIALURI).append(portal).append("/").append("g/:platform:administrators/dlp-quarantine").toString();
```

Comment:
Can you explain what is it for ?

Reply:
This is for redirect admin to the quarantine page from the mail template

Reply:
Why we need to redirect him on login url ? 
What is the behaviour if you put the url of the quarantine page without login ?

Reply:
The same behaviour !
Updated.